src/components/NavigationBar/NavigationBar.jsx

```diff
@@ -16,7 +17,12 @@ import Theme from '../../styles/themes/Theme';
 import imgLogo from '../../assets/logo.png';
 import { useStyles } from './NavigationBarStyle';
 
-const pages = ['About Us', 'Contact', 'Sign Up', 'Log In'];
+const pages = [
+  { title: 'About Us', pathname: 'AboutUs' },
+  { title: 'Contact', pathname: 'Contact' },
+  { title: 'Sign Up', pathname: 'SignUp' },
+  { title: 'Log In', pathname: 'Login' },
```

Comment:
in src/ create folder named "constants". Inside that folder create file paths.js and inside 
```js
export const paths = {
  home: '/',
  login: '/login',
  signUp: '/signUp',
  doctorVisit: '/doctor-visit'
  itp...
}
```

and there import taht file and write:

```js
{title: 'About Us', pathname: patsh.aboutUs}
```

and in App.jsx the same

why? because, when someone change path in App.jsx, then navigation stop working, but when you use variables, one change in paths object is both the same for App and Navigation and other places in code whent paths is used

Reply:
@TenGosc007 Could you look at this navigation, that I understood correctly using https://mui.com/guides/routing/